src/parsec_service/listeners.md

```diff
@@ -0,0 +1,12 @@
+# Listeners
+
+## Domain Socket Listener
+
+This listener communicates with its client using a [Unix domain
+socket](https://en.wikipedia.org/wiki/Unix_domain_socket).
+
+When launching directly from terminal, the socket path is fixed to `/tmp/security-daemon-socket`.
```

Comment:
The systemd daemon is also launched from the terminal. Maybe it's more clear to say something like 
> When running the service executable directly, ...

Reply:
Yes, makes more sense!